scripts/check_frontend_coverage.py

```diff
@@ -45,7 +45,7 @@
     'answer-group-editor.directive.ts',
     'App.ts',
     'audio-bar.directive.ts',
-    'audio-file-uploader.directive.ts',
+    'audio-file-uploader.component.ts',
```

Comment:
I see a new test file `audio-file-uploader.component.spec.ts`, are we expecting to remove this line?

Reply:
But the spec doesnot provide 100% coverage.